README.md

```diff
@@ -62,6 +62,7 @@ These sections show how to use the SDK to perform API management functions. Befo
 9. [Manage JWTs](#manage-jwts)
 10. [Search Audit](#search-audit)
 11. [Embedded Links](#embedded-links)
+12. [Manage Authz](#manage-authz)
```

Comment:
Do we want to add the term ReBAC to menu ?

Reply:
Sure